web/themes/custom/vote_gov/templates/field/field--block-content--field-heading--registration-form-selector.html.twig

```diff
@@ -0,0 +1,53 @@
+{#
+/**
+ * @file
+ * Theme override for a field.
+ *
+ */
+#}
+
+{% if element.0["#format"] is not empty %}
+  {% set text_format = element.0["#format"] %}
+{% endif %}
+
+{% set classes = [
+  'field',
+  'field--name-' ~ field_name|clean_class,
+  'field--type-' ~ field_type|clean_class,
+  'field--label-' ~ label_display,
+  label_display == 'inline' ? 'clearfix',
+  text_format in ['full_html', 'basic_html'] ? 'is-wysiwyg'
+] %}
+
+{% set title_classes = [
+  'field__label',
+  label_display == 'visually_hidden' ? 'visually-hidden',
+] %}
+
+{% if label_hidden %}
+  {% if multiple %}
+    <div{{ attributes.addClass(classes, 'field__items') }}>
+      {% for item in items %}
+        <div{{ item.attributes.addClass('field__item') }}>{{ item.content['#context'].value | render ({ '@state_name': pagetitle | render }) }}</div>
```

Comment:
The @state_name is not replaced within the string coming from item.content. How should we handle this?